static/docs/commands-reference/checkout.md

```diff
@@ -194,8 +194,11 @@ Let's check the `model.pkl` entry in `train.dvc` again:
 
 ```yaml
 outs:
-  md5: a66489653d1b6a8ba989799367b32c43
-  path: model.pkl
+  - cache: true
+    md5: a66489653d1b6a8ba989799367b32c43
+    metric: false
+    path: model.pkl
```

Comment:
path should go level up

Reply:
in the .dvc file, path option is after metric. is it still required to move it further?

Reply:
yep, my bad, good fix. disregard my comment, please.